pyproject.toml

```diff
@@ -109,6 +109,10 @@ responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
 lark = "^1.1.5"
 pytest-mock  = "^3.10.0"
+atlassian-python-api = "^3.36.0"
```

Comment:
@eyurtsev so is right way to do this now to drop these from dependencies and just mark unit tests with pytest.mark.requires